IdentityCore/tests/MSIDKeyOperationUtilTest.m

```diff
@@ -0,0 +1,251 @@
+    
+    __auto_type data = [NSData hexStringToData:hexString];
+    
+    NSDictionary *attributes = @{ (id)kSecAttrKeyType: (id)kSecAttrKeyTypeRSA, (id)kSecAttrKeySizeInBits: @2048, (id)kSecAttrKeyClass: (id)kSecAttrKeyClassPrivate };
+    
+    SecKeyRef signingKey = NULL;
+    if (@available(iOS 10.0, *))
+    {
+        signingKey = SecKeyCreateWithData((__bridge CFDataRef)data, (__bridge CFDictionaryRef)attributes, NULL);
+    }
+    
+    self.rsaPrivateKey = signingKey;
```

Comment:
nit might want to add retain/release logic for the keys, althought it might not be as important since you are doing it in the test app.

Reply:
Good catch. Updated test case